src/components/Card.js

```diff
@@ -59,6 +68,14 @@ const KudosContent = styled.p`
 `
 const KudosAuthor = styled.p`
   font-size: 12px;
-  margin-top: 20px;
 `
+const Avatar = styled.div`
+  height: ${props => props.mini ? '20px' : '25px'};
```

Comment:
I like looking at styled components 👍

Reply:
On the other hand I am wondering ... if `mini` is the only prop, what do you think about:

```
const Avatar = styled.div`
  height: 25px;
  width: 25px;
  margin: 0;
  border-radius: 50%;
  background-image: url(${props => props.image});
  background-size: cover;
`

const AvatarMini  = styled.div`
  height: 20px;
  width: 20px;
  margin: 0 2.5px;
`
```

pros:
- having props & trenary oparators inside the styled component are hard to read and IDE cannot leverage of CSS-syntax
- `<Avatar big .../>`would introduce quite a high complexity (i.e. prop flag that is mixing state) 

cons:
- if there will be some connected props (example: `mini` and `charming`) which would imply on single css property (example: `color: mini & charming ? 'pink' : (charming ? : 'starblue' : 'black')`) this approach may be hard to maintain. On the other hand, that kind of mix should worry us 

I would recommend keeping `props` as the last-resort. Honestly, `Avatar` being is something small so we can keep the `props` approach for now on if you insist ; )

Just for learning purposes, we may have much more flexible solution (but much more complicated as well)

```
// assume that `type` is one of `mini | medium | huge`
const avatar = {
 size: (type) => {
  switch(type) {
   case 'mini':
     return css`
       width: 20px;
        ...
     `
   case '...': ...
  }
 }
}
```
then:
```
const Avatar = styled.div`
  // size
  ${avatar.size(${p => p.type})};
  // other props
 ...
`
```

Reply:
Really "trenary oparators inside the styled component are hard to read"? 😮 💔

Reply:
` IDE cannot leverage of CSS-syntax` is more appealing, true ; )
`trenary oparators inside the styled component are hard to read` - yes, everything beyond simple form and simple conditiom

Reply:
I made a mistake in the code:

should be: `const AvatarMini  = styled(Avatar)`
was: `const AvatarMini = styled.div`

Reply:
Ok, I agree with that! Extended styled-component looks more professional :D

Reply:
Refactored ✅